test/util.test/Parsing.test.ts

```diff
@@ -54,4 +54,4 @@ describe('util/parsing', () => {
       });
     });
   });
-});
+});
```

Comment:
Changes here can also be undone.

Also, it looks like your editor removes empty lines at the end of files, while these are typically good to have. Maybe a config option somewhere in your editor?